packages/esm-patient-labs-app/src/lab-orders/add-lab-order/add-lab-order.scss

```diff
@@ -3,45 +3,6 @@
 @import '~@openmrs/esm-styleguide/src/vars';
 @import "../../root";
 
-.orderForm {
```

Comment:
Some of this got split out into `lab-order-form.scss` and `test-type-search.scss`